### ISSUE TITLE
pnnx转换 in_channels 为 1 的 Conv2d 层时出现问题

### ISSUE THREAD

**Issue by Mo-yun** [pnnx/pnnx#52]
模型代码如下：
import torch
from torch import nn

m = nn.Sequential(nn.Conv2d(1, 1, (3, 3), (1, 1)), nn.Conv2d(1, 2, (3, 3), (1, 1)))
mt = torch.jit.trace(m, torch.randn(1, 1, 5, 513))
mt.save("test.pt")

转换命令为：
pnnx.exe .\test.pt inputshape="[1, 1, 5, 513]" optlevel=2

加载模型报错：
ModelBin read quantization_value failed 42
layer load_model 2 conv_1 failed

**Comment by Mo-yun** [comment]
如下的模型无法正常inference
class model(nn.Module):
    def __init__(self) -> None:
        super().__init__()

    def forward(self, x, y):
        T0, T1 = x.shape[2], y.shape[2]
        c = torch.cat((x, y), dim=2)
        x = c[:, :, 0: T1, :]
        y = c[:, :, T1: T0 + T1, :]
        return x, y

**Comment by Mo-yun** [comment]
如下的模型无法正常inference
    class model(nn.Module):
        def __init__(self) -> None:
            super().__init__()

        def forward(self, x, y):
            T0, T1 = x.shape[2], y.shape[2]
            c = torch.cat((x, y), dim=2)
            x = c[:, :, 0: T1, :]
            y = c[:, :, T1: T0 + T1, :]
            return x, y

**Comment by Mo-yun** [comment]
如下模型无法正常inference
`
    class model(nn.Module):
        def __init__(self) -> None:
            super().__init__()

        def forward(self, x, y):
            T0, T1 = x.shape[2], y.shape[2]
            c = torch.cat((x, y), dim=2)
            x = c[:, :, 0: T1, :]
            y = c[:, :, T1: T0 + T1, :]
            return x, y
`

**Comment by Mo-yun** [comment]
如下的模型在转换为ncnn后无法正常inference
```python
class model(nn.Module):
    def __init__(self) -> None:
        super().__init__()

    def forward(self, x, y):
        T0, T1 = x.shape[2], y.shape[2]
        c = torch.cat((x, y), dim=2)
        x = c[:, :, 0: T1, :]
        y = c[:, :, T1: T0 + T1, :]
        return x, y
```

**Comment by csukuangfj** [comment]
> pnnx.exe .\test.pt inputshape="[1, 1, 5, 513]" optlevel=2

你用个大一点的 H, 比如，把 5 换成 50 试试？

**Comment by Mo-yun** [comment]
> > pnnx.exe .\test.pt inputshape="[1, 1, 5, 513]" optlevel=2
> 
> 你用个大一点的 H, 比如，把 5 换成 50 试试？

还是不行

**Comment by nihui** [comment]
最新版本已修复
